390/sophie_li.Rmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "MP2 Visualization"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(tidyverse)
+```
+
+```{r}
```

Comment:
hey sophie, here is a code review! The output is IMO a little jarring with all the warning messages. How about setting `warning = FALSE` to make the output html a little cleaner